feature/aft/aft_base/otg_tests/aft_base/README.md

```diff
@@ -0,0 +1,140 @@
+
+
+## AFT-1.1.4: AFT Base Link Up scenario 2
+
+Bring up both link between ATE:port1,port2 and DUT:port1,port2
+
+Verifications
+
+*   BGP routes advertised from ATE:port1,port2 must have 2 next hops.
+*   IS-IS routes advertised from ATE:port3 must have one next hop.
+*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+
+
+## OpenConfig Path and RPC Coverage
+
+The below yaml defines the OC paths intended to be covered by this test.  OC paths used for test setup are not listed here.
+
+```yaml
+paths:
+  ## Config Paths ##
+
+  ## State Paths ##
+ 
+  /network-instances/network-instance/afts/ethernet/mac-entry/state/next-hop-group:
+  /network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/next-hop-group:
+  /network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/origin-protocol:
+  /network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/prefix:
+  /network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/next-hop-group:
+  /network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/origin-protocol:
+  /network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/prefix:
+  /network-instances/network-instance/afts/aft-summaries/ipv4-unicast/protocols/protocol/state/origin-protocol:
+  /network-instances/network-instance/afts/aft-summaries/ipv4-unicast/protocols/protocol/state/counters/aft-entries:
+  /network-instances/network-instance/afts/aft-summaries/ipv6-unicast/protocols/protocol/state/origin-protocol:
+  /network-instances/network-instance/afts/aft-summaries/ipv6-unicast/protocols/protocol/state/counters/aft-entries:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/id:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/index:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/index:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/weight:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/backup-next-hop-group:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/id:
+  /network-instances/network-instance/afts/next-hops/next-hop/index:
+  /network-instances/network-instance/afts/next-hops/next-hop/interface-ref/state/interface:
+  /network-instances/network-instance/afts/next-hops/next-hop/interface-ref/state/subinterface:
+  /network-instances/network-instance/afts/next-hops/next-hop/state/encapsulate-header:
+  /network-instances/network-instance/afts/next-hops/next-hop/state/index:
+  /network-instances/network-instance/afts/next-hops/next-hop/state/ip-address:
+  /network-instances/network-instance/afts/next-hops/next-hop/state/mac-address:
+  /network-instances/network-instance/afts/next-hops/next-hop/state/origin-protocol:
+  /network-instances/network-instance/afts/state-synced/state/ipv4-unicast:
```

Comment:
Do we call out these `state-synced` leaves as being tested here?

Reply:
Kindly clarify

Reply:
I did not see which test above checked this path. Could you mention how it's being used?

Reply:
updated